core/templates/dev/head/pages/review-test-page/review-test-page.directive.ts

```diff
@@ -68,10 +78,30 @@ oppia.directive('reviewTestPage', ['UrlInterpolationService', function(
                 result.data.skill_descriptions[skillId]);
             }
             var questionPlayerConfig = {
+              resultActionButtons: [
+                {
+                  type: 'BOOST_SCORE',
+                  text: 'Boost My Score'
+                },
+                {
+                  type: 'RETRY_SESSION',
+                  text: 'Retry Test',
+                  url: reviewTestsUrl
+                },
+                {
+                  type: 'DASHBOARD',
+                  text: 'Return To Story',
+                  url: storyViewerUrl
+                }
+              ],
               skillList: skillIdList,
               skillDescriptions: skillDescriptions,
               questionCount: ReviewTestEngineService.getReviewTestQuestionCount(
-                skillIdList.length)
+                skillIdList.length),
+              questionPlayerMode: {
+                modeType: 'PASS_FAIL',
+                passCutoff: 0.75
```

Comment:
This should be stored in a constant I think (if you plan to use this elsewhere too).

Reply:
It will be referred to as quesionPlayerMode.passCutoff elsewhere, so I don't think we need a constant for this one.